src/commonMain/kotlin/app/revanced/library/ApkUtils.kt

```diff
@@ -53,7 +53,7 @@ object ApkUtils {
     fun PatcherResult.applyTo(apkFile: File) {
         ZFile.openReadWrite(apkFile, zFileOptions).use { targetApkZFile ->
             dexFiles.forEach { dexFile ->
-                targetApkZFile.add(dexFile.name, dexFile.stream)
+                dexFile.stream.use { targetApkZFile.add(dexFile.name, it) }
```

Comment:
I think doing it like this is a little more readable as you dont nest business logic under the scope of streams that doesnt use the streams
```suggestion
                targetApkZFile.add(dexFile.name, dexFile.stream)
                dexFile.stream.close()
```